client/src/lib/components/sandbox/play/Arena.svelte

```diff
@@ -61,4 +97,19 @@
 			Player 2: {truncateMinaPublicKey(players[1])}
 		</div>
 	</div>
+	<span id="piece-hover-tooltip">
+		{#if hoveredPiece}
+			{@const hoveredUnit = hoveredPiece.playerUnit.unit}
+			<div>{hoveredPiece.playerUnit.name} ({hoveredUnit.name})</div>
+			<div>HP: {hoveredPiece.health}/{hoveredUnit.maxHealth}</div>
+			<div>Speed: {hoveredUnit.movementSpeed}</div>
+			<div>Armor: {hoveredUnit.armorSaveRoll}+</div>
+			{#if hoveredUnit.rangedNumAttacks}
+				<div>Missile Range: {hoveredUnit.rangedRange}</div>
+			{/if}
+			{#if selectedPiece}
+				<div></div>
+			{/if}
```

Comment:
I had to include some reference to `selectedPiece` in the HTML here in order for a change to `selectedPiece` to trigger a redraw of the component. Without this here, only a change to `hoveredPiece` triggers a redraw.

Once I move on to issuing move orders with the mouse, a change of `selectedPiece` will result in us showing/hiding the move range circle for that piece, so this won't be an issue then and this bit here can be removed.